metric_tank/query_engine.go

```diff
@@ -57,16 +57,17 @@ func alignRequests(reqs []Req, aggSettings []aggSetting) ([]Req, error) {
 	}
 	tsRange := (reqs[0].to - reqs[0].from)
 
+	// note: not all series necessarily have the same raw settings, will be fixed later
```

Comment:
are you saying that "fixed later" means "further down" and not "some time in the furture"